openshift/4.0/templates/backup/README.md

```diff
@@ -90,15 +134,10 @@ MEMORY_LIMIT=1Gi
 
 Create the deployment and save the template.
 
-```bash
-oc process -f deploy-proxy-caddy.yaml --param-file=deploy-proxy-caddy.dev.env | oc create --save-config=true -f -
-```
-
-Ensure that the deployment config for `pims-database-${environment}` contains a volume mapping for the backup volume. This will need to be added after the backup volume has been created for the first time in the target environment. There is a commented out section in mssql-deploy with this configuration, or you can complete this task in the GUI by mapping the backup volume to path `/backups`.
+## OpenShift CronJobs
 
-Tag the latest `mssql-backup` image for the appropriate environment (i.e. `dev`).
-This will trigger the deployment configuration to deploy the pod.
+OpenShift 4.0 supports cronjobs natively.
```

Comment:
this may be something they are already working on in the backup container repo? let me check.

Reply:
Nope.

Reply:
arguably there are other, higher priority issues in that DB to work on. But maybe if I take on a few of those for Wade he could work on the above.